lib/money/deprecations.rb

```diff
@@ -7,7 +7,10 @@ def self.active_support_deprecator
 
   def self.deprecate(message)
     if ACTIVE_SUPPORT_DEFINED
-      active_support_deprecator.warn("[Shopify/Money] #{message}\n")
+      external_callstack = caller_locations.reject do |location|
+        location.to_s.include?('gems/money')
```

Comment:
I'm a bit worried that this could produce a huge amount of logs

Reply:
Rails will only take the first item from the caller_locations array. This is similar to the rails default `caller_locations(2)` which just sets the start at the second element (http://www.rubydoc.info/stdlib/core/2.0.0/Kernel%3Acaller_locations) 
```
[1] pry(Money)> caller_locations.size
=> 115
[2] pry(Money)> caller_locations.first
=> "/Users/michaelelfassy/.gem/ruby/2.3.3/gems/pry-0.10.4/lib/pry/pry_instance.rb:355:in `eval'"
[3] pry(Money)> caller_locations(2).size
=> 114
[4] pry(Money)> caller_locations(2).first
=> "/Users/michaelelfassy/.gem/ruby/2.3.3/gems/pry-0.10.4/lib/pry/pry_instance.rb:355:in `evaluate_ruby'"
```